content/docs/contributing/to-docs/submit-a-pr.md

```diff
@@ -48,6 +48,10 @@ For complete reference on the `commit` command, refer to the [Git commit referen
 
 For more information on staging and committing files with Git, refer to [Git Basics - Recording Changes to the Repository](https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository).
 
+{{< important >}}
+If you used the web UI to make your commit vs the commandline and your PR fails the DCO check, you may need to use `username@users.noreply.github.com` as the email in the `Signed-off-by:` commit message line.
+{{< /important >}}
+

```

Comment:
Good idea to add more information about DCO here. The GitHub "noreply" email is only one possibility, though. I think with a little more research on what email addresses can be used on web-based commits, this note can be made more precise. This GitHub doc looks like it would help: https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-email-preferences/setting-your-commit-email-address

Reply:
Good call I'll put in a link to that as a more information while adding the information about newer email addresses needing the 7 digits. Will get this back up in a bit